Description: Comment out editCellAt method calls
 Comment out editCellAt method calls as it's not implemented in Classpath.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/hostmon/KnownHostTable.java
===================================================================
--- airport-utils-2.orig/hostmon/KnownHostTable.java	2010-03-11 20:55:56.652753360 +0100
+++ airport-utils-2/hostmon/KnownHostTable.java	2010-03-11 20:58:12.725753655 +0100
@@ -287,7 +287,7 @@
 		catch(ValueFormatException e)
 		{
 		    // set the editing to the mac address in the row that caused the problem
-		    table.editCellAt(i,1);
+		    //table.editCellAt(i,1);
 			
 			// rethrow the exception so can report it
 			throw e;
